crypto/mem.c

```diff
@@ -57,10 +57,11 @@
 #include <ring-core/mem.h>
 #include "internal.h"
 
-int OPENSSL_memcmp(const void *av, const void *bv, size_t len) {
+int CRYPTO_memcmp(const void *in_a, const void *in_b, size_t len) {
+  const aliasing_uint8_t *a = in_a;
+  const aliasing_uint8_t *b = in_b;
   uint8_t x = 0;
-  const aliasing_uint8_t *a = av;
-  const aliasing_uint8_t *b = bv;
+
```

Comment:
This is just making the function structure the same as BoringSSL's version. The only difference is the use of `aliasing_uint8_t` instead of `uint8_t`.